pkg/mounter/mounter-rclone.go

```diff
@@ -111,6 +112,9 @@ func newRcloneMounter(secretMap map[string]string, mountOptions []string) (Mount
 	if val, check = secretMap["secretKey"]; check {
 		secretKey = val
 	}
+	if val, check = secretMap["kp-root-key-crn"]; check {
+		mounter.kpRootKeyCrn = val
+	}
```

Comment:
@Bhagyashreek8  In test results for s3fs you have provided the screen shot for bucket details with "Associated key management services". Could you pls provide same for a new bucket created from rclone.  I just want to make sure this works as expected as I dont see mount options passed in rclone mounter.